tests/integration/mutation/create/crdt/pncounter_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package create
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestMutationCreate_PNCounter_NoError(t *testing.T) {
```

Comment:
todo: word "Mutation" is redundant here and doesn't add any clarity.

I also think that "PNCounter" should in the first part of the test name. The middle part is for precondition.
suggestions: `TestPNCounterCreate_ValidInput_NoError`, `TestPNCounterCreate_IfValidInput_ShouldSucceedAndCanQuery`

Reply:
I'm trying to be consistent with the naming so that it matched the pattern of the tests in the parent packages. I understand your concern though and I'll apply some changes.

Reply:
✅

Reply:
Just noting that naming the tests in a consistent pattern is really useful when running a sub-set.  For example `go test ./... -run TestMutation` would run all the mutation tests.  I have quite a strong preference for preserving the original name.